qa-tests-backend/src/test/groovy/IntegrationsTest.groovy

```diff
@@ -68,54 +72,85 @@ class IntegrationsTest extends BaseSpecification {
         DEPLOYMENTS.each { orchestrator.deleteDeployment(it) }
     }
 
+    @SuppressWarnings('LineLength')
     @Unroll
     @Category([BAT])
-    @Ignore("ROX-8113 - email tests are broken")
-    def "Verify create Email Integration (port #port, disable TLS=#disableTLS, startTLS=#startTLS)"() {
+    def "Verify create Email Integration (disableTLS=#disableTLS, startTLS=#startTLS, authenticated=#authenticated, sendCreds=#sendCreds)"() {
         given:
+        "mailserver is running"
+        def mailServer = MailServer.createMailServer(orchestrator, authenticated, !disableTLS)
+        sleep 15 * 1000 // wait 15s for service to start
```

Comment:
I'd prefer something other then a 15 second sleep but ok if you want to fix in ROX-12418